content/posts/renova-project-sasha-interview.md

```diff
@@ -0,0 +1,110 @@
+har fungerat väldigt bra för henne. Hon fick hjälp att komma in hos Renova just
+på grund av hennes intresse för närproducerad kompostjord - en förutsättning för
+ett närproducerat organiskt jordbruk. Renova Marieholm har nämligen redan en tid
+bedrivit kompostering.
+
+"Till en början gick jag bara runt och plockade ut skräp ur komposthögarna,
+området och stängslen däromkring. Jag tog skottkärran på morgonen och plockade
+så mycket skräp jag kunde. Arbetet var enahanda och nedslående på så sätt att
+det inte såg ut att göra någon skillnad, så efter en tid började det göra mig nedslagen.
+Jag insåg att skräpet kom med komposten, och att en effektiv kompostering kräver
+att det inkommande materialets kvalité granskas och säkerställs genom rensning
+redan vid källan."
+
+### En effektivare process
+
+Sasha började tala med sopåkarna som kom in med trädgårdsavfallet
+för att med deras hjälp ordna avvikelserapporter och därigenom återkoppling till
+de kunder som inte åstadkommit tillräcklig separation av återvinningsmaterialen
+(t ex genom att plast hamnat i bingen för trädgårdsavfall) och kunde på så sätt
+höja kvalitén på det inkommande materialet avsevärt.
+
+Nästa problem som Sasha uppdagade var hur gräs och blommor växte till på komposthögen.
+"Det visar ju å ena sidan att det finns mycket näring i komposten, men det är å
+andra sidan ett tecken på att uppvärmningen är otillräcklig eftersom den borde
+dödat växternas frön. Att uppnå tillräckligt hög temperatur under längre tid är
+nödvändigt för att säkerställa att inga patogener överlever till den färdiga
+produkten", säger Sasha. "Vid det här laget började jag titta på litteratur -
+det finns många exempel på storskalig kompostering framför allt ifrån USA, som
+är industriledande i denna branch.
+Den genomsnittliga genomströmmningstiden för material där är 45 till 90 dagar, vilket
+är väldigt mycket mer effektivt än de femton månader som Renovas process föreskrev."
+
+Med stöd ur litteraturen fick Sasha i det här läget klartecken för att starta upp
+den helt aeroba process med aktiv luftning som är industristandard i USA och som
+också reducerar metanproduktionen och kraftigt ökar materialgenomströmmningen.
+Pilotanläggningen hade till slut två komposthögar, där den ena byggdes på medan
+den andra låg och mognade i två veckor. Därefter vändes och återfuktades den, och
+efter fyra veckor hade den stabiliserats och behövde bara mogna ytterligare för
+att skapa god matjord. Detta var väldigt lovande; temperaturerna var nu tillräckligt
+höga för att desinfektera materialet, men de svängde mycket över tid.
+
+"Jag utgick ifrån Robert Rynks 'The composting handbook' och använde standardvärden",
+berättar Sasha. "Det var en fantastisk resurs för mig; jag hade inga möjligheter
+att skaffa konsulttimmar hos experter eller ingenjörer för att få ordning på processen
+eftersom det var en liten försöksverksamhet utanför Renovas egentliga verksamhet",
+fortsätter hon.
+
+### IFK till undsättning
```

Comment:
"IFK till undsättning" gav Sasha känsla av att vara "damsel in distress"...
Kan man tona ner den lite?

Reply:
Absolut. Jag provar något annat. Bra poäng, det är inte snyggt i tonen.